tools/cli/go-whisk/whisk/client.go

```diff
@@ -126,6 +127,10 @@ func NewClient(httpClient *http.Client, config *Config) (*Client, error) {
         config.Version = "v1"
     }
 
+    if len(config.CLIVersion) == 0 {
+        config.CLIVersion = "not_set"
+    }
+
```

Comment:
In your testing did you find this default setting to be needed?  i was thinking this check/set is not necessary due to the way the CLI version string is initialized